index.js

```diff
@@ -1,25 +1,26 @@
-const express = require('express')
-const app = express()
-const auth = require('./auth')
-const bodyParser = require('body-parser')
-const logger = require('morgan')
+const express = require('express');
 
-app.use(bodyParser.json())
+const app = express();
+const bodyParser = require('body-parser');
+const logger = require('morgan');
 
-if(process.env.NODE_ENV != 'test'){
-    app.use(logger('dev'))
+app.use(bodyParser.json());
+
+if (process.env.NODE_ENV !== 'test') {
+    app.use(logger('dev'));
 }
 
 app.get('/ping', (req, res) => {
     res.json({
-        message: 'pong'
-    })
-})
-app.use('/users', require('./routes/user-routes'))
-app.use('/posts', require('./routes/post-routes'))
-app.use('/follow', require('./routes/follow-routes'))
-
-app.listen(process.env.PORT || 8080)
+        message: 'pong',
```

Comment:
why is there a comma?

Reply:
I like that, good habit :)

Reply:
Part of the coding style + if we want to add more properties, it'll show as an addition instead of a change

Reply:
Part of the coding style + if we want to add more properties, it'll show as an addition instead of a change